plugin/src/main/java/net/dzikoysk/funnyguilds/listener/region/GuildHeartProtectionHandler.java

```diff
@@ -62,13 +63,13 @@ public static boolean isGuildHeart(PluginConfiguration config, Block block) {
             return false;
         }
 
-        Location loc = block.getLocation();
-        Region region = RegionUtils.getAt(loc);
+        Location blockLocation = block.getLocation();
 
-        if (region == null) {
+        Option<Region> regionOption = FunnyGuilds.getInstance().getRegionManager().findRegionAtLocation(blockLocation);
+        if (regionOption.isEmpty()) {
```

Comment:
można coś takiego:
return FunnyGuilds.getInstance().getRegionManager().findRegionAtLocation(blockLocation)
.map(region -> blockLocation.equals(region.getHeart()))
.orElseGet(false);

ogólnie to ta metoda chyba nie powinna się znajdować tutaj, jest mocno zależna od RegionManager więc można tam ją dać